### ISSUE TITLE
Missing coastline and bathymetry

### ISSUE THREAD

**Issue by kamwal** [euroargodev/User-Acceptance-Test-Python-version-of-the-OWC-tool#33]
I have found that for some floats Python code is not displying coastline contours of continents and bathymetry in trajectory plot.

Left plot -Python verion; Right plot- Matlab version
![image](https://user-images.githubusercontent.com/57108044/86767177-7f577000-c043-11ea-8c2d-b6df48574cc5.png)

**Comment by edsmall-bodc** [comment]
Odd! Also looks like the points should be a little smaller?

**Comment by kamwal** [comment]
Yes, that is also true. It would be good to have them a bit smaller.

**Comment by edsmall-bodc** [comment]
After a quick investigation with @kamwal it became obvious that some floats operate on 0-360, not -180 to 180. This could also be linked to the problems we are having [HERE](https://github.com/euroargodev/User-Acceptance-Test-Python-version-of-the-OWC-tool/issues/9), which I will investigate further.

A quick and easy fix for the plotting would be that if any of the points exceed 180 then to -180 from all longitudes. This should solve the issue.

**Comment by kamwal** [comment]
Yes I agree, there is also some more issues with longitudes

Left plot  from Python; Right plot  from Matlab 

![image](https://user-images.githubusercontent.com/57108044/86799280-afb20500-c069-11ea-92a5-54aacad9e74a.png)

**Comment by kamwal** [comment]
I am including data to these floats:
[3901985.zip](https://github.com/euroargodev/User-Acceptance-Test-Python-version-of-the-OWC-tool/files/4917545/3901985.zip)
[4901464.zip](https://github.com/euroargodev/User-Acceptance-Test-Python-version-of-the-OWC-tool/files/4917546/4901464.zip)
[6901121.zip](https://github.com/euroargodev/User-Acceptance-Test-Python-version-of-the-OWC-tool/files/4917547/6901121.zip)
[3900560.zip](https://github.com/euroargodev/User-Acceptance-Test-Python-version-of-the-OWC-tool/files/4917548/3900560.zip)
[3901531.zip](https://github.com/euroargodev/User-Acceptance-Test-Python-version-of-the-OWC-tool/files/4917549/3901531.zip)

**Comment by edsmall-bodc** [comment]
This is fixed by [this PR](https://github.com/euroargodev/argodmqc_owc/pull/16) but we need to adjust the CI pipeline to use conda to install cartopy

**Comment by kamwal** [comment]
@gmaze Please, could you change the pipeline to use conda to install cartopy and harmonica?
